--- getpic.c	2012-10-22 17:48:16.000000000 +0200
+++ /opt/liquid_feedback_frontend-v2.1.1/fastpath/getpic.c	2012-10-15 22:26:45.000000000 +0200
@@ -8,11 +8,11 @@
 #include <unistd.h>
 
 #ifndef GETPIC_CONNINFO
-#define GETPIC_CONNINFO "dbname=liquid_feedback"
+#define GETPIC_CONNINFO "dbname=liquid_feedback_lsa"
 #endif
 
 #ifndef GETPIC_DEFAULT_AVATAR
-#define GETPIC_DEFAULT_AVATAR "/opt/liquid_feedback_testing/app/static/avatar.jpg"
+#define GETPIC_DEFAULT_AVATAR "/opt/liquid_feedback_frontend/static/avatar.jpg"
 #endif
 
 int main(int argc, const char * const *argv) {
